### PR TITLE
Reformatting of files and functions

### PR DESCRIPTION
**Pull request by JoshuaSlui**
This PR brings a lot of code changes, as it now subdivides all utility feats under 3 files:
- database
- Social
- Data

Database handles all functions for database inquiries
Social handles all functions for social commands
Data handles all data, such as Emotes, Links and Colors

Additionally, this update adds a global check to handle disabled commands.
All commands have also been changed to use this global check, and to use the database functions.

I have also ensured that most, if not all of it is PEP8-valid and have changed all class names to follow CamelCase conventions.

### CODE REVIEW

File: cogs/events/errors.py [hunk 1]
```diff
@@ -17,21 +15,27 @@ async def on_application_command_error(self, ctx, err):
...
 
         if isinstance(err, discord.NotFound):
-            return await ctx.respond("I could not find the argument you have provided.", ephemeral=True)
+            return await ctx.respond(f"{Emotes.confused} I could not find the argument you have provided.", ephemeral=True)
+
+        else:
+            embed = discord.Embed(colour=Colors.red)
+            embed.description = f"You can join our support discord [here]({Links.discord})"
+            print(traceback.format_exception(err))
+            return await ctx.respond(f"{Emotes.confused} An error has occurred. Please report this to my developers.", embed=embed)
```
Comment (ToothyDev) [inline cogs/events/errors.py:37]: I originally copied this over from the prefix commands error handler, and I am unsure which situations trigger this, but I'd like to know

File: utilities/data.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+class Emotes:
+    bankCard = '<a:MoneyCard2:1103307033941385247>'
+    cash = '<:cash:1103307142410272768>'
+    crossmark = '<:Cross:845654734916681749>'
+    checkmark = '<:Check:909435999216623636>'
+    confused = '<a:loading:908629364378325023>'
+
+class Links:
+    discord = 'https://discord.gg/qWyNwSx2rx'
```
Comment (ToothyDev) [inline utilities/data.py]: No newline at end of file

**Review by ToothyDev — DISMISSED** [review]
Looks like a good and comprehensive reformat of much of the code, and logical re-moves of certain functions that were previously misplaced 👍

File: cogs/events/errors.py (hunk 1 above)
Comment (JoshuaSlui) [reply cogs/events/errors.py:37]: In the else clause?

File: utilities/data.py (hunk 2 above)
Comment (JoshuaSlui) [reply utilities/data.py]: _nitpicking, you know what i said about that_

File: cogs/events/errors.py (hunk 1 above)
Comment (ToothyDev) [reply cogs/events/errors.py:37]: My bad, I mistook that for being entirely under the isinstance block - I was referring to the isinstance block which seems to catch arguments that weren't found, and I was unsure whether that's possible with slash commands

Comment (JoshuaSlui) [reply cogs/events/errors.py:37]: https://discordpy.readthedocs.io/en/stable/api.html?highlight=notfound#discord.NotFound
This block was not added in this PR, i only added the emote to there.

To me, it seems to be necessary to keep this.

**Review by ToothyDev — APPROVED** [review]

**Pull request merged**